eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -401,6 +384,40 @@ steps:
     dropMetadataContainerName: 'DropMetadata-ProfilingInputs'
   condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
 
+- task: MicroBuildBuildVSBootstrapper@2
+  displayName: 'Build a Visual Studio bootstrapper for tests'
+  inputs:
+    channelName: "$(VsTargetChannelForTests)"
+    vsMajorVersion: "$(VsTargetMajorVersion)"
+    manifests: '$(Build.Repository.LocalPath)\artifacts\VS15\Microsoft.VisualStudio.NuGet.Core.vsman'
+    outputFolder: '$(Build.Repository.LocalPath)\artifacts\VS15'
+  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'))"
```

Comment:
The size of the bootstrapper isn't huge, but it is megs. I'm thinking about the amount of storage that we'll use for all our retained builds if we start doubling the number of bootstrappers we have.  Although honestly, we should probably move the bootstrapper out of the VS15 directory and put it somewhere that isn't retained.

What do you think of another idea, where the PR/official pipelines still only builds one bootstrapper, but the OptProf pipeline creates its own bootstrapper? OptProf pipeline runs aren't retained for long, so we'll never need to worry about long term storage costs.